src/js/views/Navigation.vue

```diff
@@ -131,6 +131,7 @@ export default {
 		...mapState({
 			isPollCreationAllowed: (state) => state.polls.meta.permissions.pollCreationAllowed,
```

Comment:
@come-nc 
Watch the changed structure. For the stable-6 branch, the mapping should not chbe changed. (`meta.*` does not exist there in the store.)